tests/old/config/test_config.py

```diff
@@ -120,10 +120,6 @@ def test_get_config_form_overview(self):
         response = self.client_manager.get('/config/')
         self.assertRedirects(response=response, expected_url='/config/general/',
                              status_code=302, target_status_code=200)
-        bad_client = Client()
-        response = bad_client.get('/config/', follow=True)
-        self.assertRedirects(response=response, expected_url='/login/?next=/config/general/',
```

Comment:
It is a Feature of the PermissionMixin (which is used by this view) not to allow users which unprivileged permissions to see the page. This is not a feature of the config app.

If we change the behaviour of the PermissionMixin this test should not fail. For example if we change the statuscode to 403 instead of 302